test/wpool_process_SUITE.erl

```diff
@@ -79,7 +81,7 @@ init_timeout(_Config) ->
   {ok, Pid} =
     wpool_process:start_link(?MODULE, echo_server, {ok, state, 0}, []),
   timer:sleep(1),
-  timeout = wpool_process:call(?MODULE, state, 5000),
+  timeout = get_state(?MODULE),
```

Comment:
I find it hard to understand why this change works. Could you explain it to me?

Reply:
I just added a proper test and docs on 9ce28d5.
Thanks for the review :)